src/commands/register.ts

```diff
@@ -113,8 +117,16 @@ export default class ComponentRegister extends Command {
           delete service.build.args;
         }
 
+        let platforms: string[];
+        try {
+          platforms = DockerBuildXUtils.convertToBuildxPlatforms(flags['platform']);
+        } catch (err: any) {
+          this.log('Failed to convert platform argument to docker buildx platforms. Please provide the correct platform(s).');
```

Comment:
No need for this message the error you throw is descript enough.

Reply:
Message was removed